spec/support/coverage_setup.rb

```diff
@@ -0,0 +1,14 @@
+require 'simplecov'
+
+# Save to CircleCI's artifacts directory if we're on CircleCI
+if ENV['CIRCLE_ARTIFACTS']
+  dir = File.join(ENV['CIRCLE_ARTIFACTS'], "coverage")
+  SimpleCov.coverage_dir(dir)
+end
+
+# Evil options :D
+# SimpleCov.minimum_coverage 99
+# SimpleCov.minimum_coverage_by_file 90
+# SimpleCov.refuse_coverage_drop
+
+SimpleCov.start
```

Comment:
Have you tried running only feature specs to see if SimpleCov works for feature specs? It should be started on all processes running the application (i.e. the commands we launch when running feature specs).